dbtmetabase/models/interface.py

```diff
@@ -19,8 +23,8 @@ def __init__(
         self,
         database: str,
         host: str,
-        user: str,
-        password: str,
+        user: Optional[str] = None,
+        password: Optional[str] = None,
```

Comment:
These type changes must be propagated down to `MetabaseClient`, that's why your CI is failing

Reply:
updated, should passed the CI now